frontend/src/model/GameMode.ts

```diff
@@ -0,0 +1,20 @@
+import Settings, { boardSize, tileAmount, turnLength } from './Settings';
+
+type mode = 'Classic' | 'Fast' | 'Custom';
```

Comment:
There is a convention in typescript that types should be in PascalCase - please adjust.